src/applications/ivc-champva/10-10D/pages/ApplicantRelationshipPage.jsx

```diff
@@ -0,0 +1,284 @@
+import React, { useState, useEffect } from 'react';
+import {
+  VaButton,
+  VaRadio,
+  VaTextInput,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import PropTypes from 'prop-types';
+
+import { applicantWording } from '../helpers/wordingCustomization';
+
+const keyname = 'applicantRelationshipToSponsor';
+
+function generateOptions({ data, pagePerItemIndex }) {
+  const currentListItem = data?.applicants?.[pagePerItemIndex];
+  const personTitle = 'Sponsor';
+  const applicant = applicantWording(currentListItem).slice(0, -3); // remove 's_
+
+  // Determine what tense/person the phrasing should be in
+  const useFirstPerson =
+    data?.certifierRole === 'applicant' && +pagePerItemIndex === 0;
+
+  // Set up grammatically appropriate articles
+  const relativeBeingVerb = `${`${
+    !useFirstPerson
+      ? `${applicant} ${data.sponsorIsDeceased ? 'was' : 'is'}`
+      : `${data.sponsorIsDeceased ? 'I was' : 'I’m'}`
+  }`}`;
+
+  const relativePossessive = `${`${
+    useFirstPerson ? 'your' : `${applicant}’s`
+  }`}`;
+
+  // Create dynamic radio labels based on above phrasing
+  const options = [
+    {
+      label: `${relativeBeingVerb} the ${personTitle}'s spouse`,
+      value: 'spouse',
+    },
+    {
+      label: `${relativeBeingVerb} the ${personTitle}'s child`,
+      value: 'child',
+    },
+    {
+      label: `${relativeBeingVerb} the ${personTitle}'s caretaker`,
+      value: 'caretaker',
+    },
+    {
+      label: `${`${
+        applicant && !useFirstPerson ? `${applicant} doesn’t` : 'We don’t'
+      }`} have a relationship that’s listed here`,
```

Comment:
Can this be refactored? It doesn't seem like doubly embedded template strings should be used here.

Reply:
Something like this should work: 

```
`${ applicant && !useFirstPerson ? applicant+" doesn't" : "we don't"} have a relationship that's listed here`
```

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15097156/9e7a9f52-b037-42b4-88da-a97e13063d31)

Reply:
Can definitely fix that! I do get a linting error when I use regular string concatenation for that middle part, but getting rid of the outer template works and it looks cleaner. Is this compromise alright?

![Screenshot 2024-02-16 at 15 58 00](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/eddf50a2-21ba-4a14-b4ef-ad532dace86a)

![Screenshot 2024-02-16 at 15 58 17](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/d7b865ff-4445-4cf5-9a6a-6d31d103492d)

Reply:
Thanks @michaelclement - looks much cleaner :+1: